spring-batch-core/src/main/java/org/springframework/batch/core/launch/support/TaskExecutorJobLauncher.java

```diff
@@ -62,9 +62,9 @@
  * @see JobRepository
  * @see TaskExecutor
  */
-public class SimpleJobLauncher implements JobLauncher, InitializingBean {
+public class TaskExecutorJobLauncher implements JobLauncher, InitializingBean {
```

Comment:
For such a widely used class, it is preferable to proceed with a deprecation in v5.0 (followed by a removal in 5.2) instead of a complete removal in one go. I will take care of this when merging the PR.